src/env.js

```diff
@@ -140,20 +143,11 @@ if (fs.existsSync(vector.configFile)) {
     }
 } else {
     console.log("Impossible to load config.yml. A default configuration file has been generated.");
+}
+
+// Appneta email sender auth hack
```

Comment:
1. why are we checking in hacks?
2. if this is a hack why are we only explaining it with 1 line?

Reply:
removed